targets/nuttx-stm32f4/README.md

```diff
@@ -30,66 +31,87 @@ jerry-nuttx
   + stlink
 ```
 
-#### 2. Adding JerryScript as an interpreter for NuttX
+#### 2. Build JerryScript for NuttX
+
+Build JerryScript with the appropriate toolchain and use NuttX as sysroot:
 
 ```
-cd apps/interpreters
-mkdir jerryscript
-cp ../../jerryscript/targets/nuttx-stm32f4/* ./jerryscript/
+# Assuming you are in jerry-nuttx folder.
+cd jerryscript
+
+tools/build.py --clean --toolchain cmake/toolchain_mcu_stm32f4.cmake --profile=es2015-subset --jerry-cmdline OFF --jerry-libc OFF --lto OFF --jerry-libm ON --all-in-one ON --mem-heap 70 --compile-flag='--sysroot=../nuttx'
 ```
 
-#### 3. Configure NuttX
+#### 3. Copy JerryScript's application files to NuttX
 
 ```
-# assuming you are in jerry-nuttx folder
+# Assuming you are in jerry-nuttx folder.
+mkdir -p apps/interpreters/jerryscript
+cp jerryscript/targets/nuttx-stm32f4/* apps/interpreters/jerryscript/
+```
+
+#### 4. Configure NuttX
+
+```
+# Assuming you are in jerry-nuttx folder.
 cd nuttx/tools
 
-# configure NuttX USB console shell
+# Configure NuttX to use USB console shell.
 ./configure.sh stm32f4discovery/usbnsh
-
-cd ..
-# might require to run "make menuconfig" twice
-make menuconfig
 ```
 
-We must set the following options:
+The configuration creates a `.config` file in the root folder of NuttX. That configuration file should be modified
+to enable JerryScript.
 
-* Change `Build Setup -> Build Host Platform` from _Windows_ to _Linux_
-* Enable `System Type -> FPU support`
-* Enable `Application Configuration -> Interpreters -> JerryScript`
+##### 4.1 Enable JerryScript without user interaction
 
-If you get `kconfig-mconf: not found` error when you run `make menuconfig` you may have to install kconfig-frontends:
+```
+# Assuming you are in nuttx folder.
+echo -e "CONFIG_JERRYSCRIPT=y\nCONFIG_JERRYSCRIPT_PRIORITY=100\nCONFIG_JERRYSCRIPT_STACKSIZE=16384 >> .config"
+```
+
+##### 4.2 Enable JerryScript using kconfig-frontend
+
+`kconfig-frontend` could be useful if there are another options that should be enabled or disabled in NuttX.
 
 ```
-# assume you are in jerry-nuttx folder
 sudo apt-get install gperf flex bison libncurses-dev
+
+# Assuming you are in jerry-nuttx folder.
 git clone https://github.com/jameswalmsley/kconfig-frontends.git
```

Comment:
As akosthekiss mentioned in #2103, the nuttx/tools repo contains its own kconfig sources. I think that cloning the `https://bitbucket.org/nuttx/tools.git` would be better.

Reply:
@robertsipka Sorry, I miss that :) I will modify the README.md file.